src/components/addNote/index.tsx

```diff
@@ -2,10 +2,26 @@ import { Button } from '@mui/material';
 import Box from '@mui/material/Box';
```

Comment:
to be consistent with the the other folder names, can you name this `AddNote`

Reply:
yes